best-practices/tagging/index.html

```diff
@@ -200,15 +216,11 @@ <h4>Emphasis</h4>
 					<pre><code class="html">&lt;p>⠠⠏⠗⠑⠎⠎ ⠞⠓⠑ &lt;em class="custom-1">⠈⠼⠂⠎⠟⠥⠁⠗⠑&lt;/em> ⠃⠥⠞⠞⠕⠝⠲&lt;/p></code></pre>
 					
 				</aside>
-<p class="ednote" title="Use markup for braille-only indicators">
-  An earlier draft identified the braille-only indicators that are used for things like emphasis. This idea was not discussed with the group and has been left out. If we decide to include it, we could use spans.
-  Questions include, should we identify each kind of indicator in a unique way (bold character, bold passage, but also underline character, underline passage, etc.)-WF
-</p>
 			</section>
 
 			<section id="figures">
 				<h4>Figures</h4>
-
+				<p>[^figure^] is used to denote images. It must also include [^figcaption^] if that image is accompanied by a caption.</p>
```

Comment:
This might be too limiting a description. A figure tag could also encapsulate a table, code example, etc. Any figure with a caption, essentially.

Reply:
Fixed through responding to James' notes.